webui/react/src/pages/Wait.tsx

```diff
@@ -55,10 +55,9 @@ const Wait: React.FC = () => {
     return () => storeDispatch({ type: StoreActionUI.ShowUIChrome });
   }, [storeDispatch]);
 
-  const handleTaskError = (err: Error) => {
```

Comment:
this is a case where we were doing `handleError` with an object literal and it was being ignored... resulting in "unknown error"

here we're solving it in two ways at once by doing this here and changing `error.ts` too... but oh well.